packages/design-system/src/components/Review/ReviewLink.test.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import ReviewLink from './ReviewLink';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+const noop = () => {};
+const text = 'link text';
+const defaultProps = {
+  onClick: noop,
+  href: 'my-href',
+};
+
+function renderLink(customProps?, children = text) {
+  const props = { ...defaultProps, ...customProps };
+  return render(<ReviewLink {...props}>{children}</ReviewLink>);
+}
+
+describe('ReviewLink', function () {
+  it('renders link', () => {
+    renderLink();
+
+    const els = screen.getAllByRole('link');
+    expect(els.length).toBe(1);
+
+    const el = els[0];
+    expect(el.textContent).toBe(text);
+  });
+
+  it('props.onClick is called with correct parameters', () => {
+    const onClick = jest.fn();
+    renderLink({ onClick });
+
+    const els = screen.getAllByRole('link');
+    expect(els.length).toBe(1);
+
+    const el = els[0];
+    expect(el.textContent).toBe(text);
+
+    userEvent.click(el);
+    expect(onClick).toHaveBeenCalledTimes(1);
```

Comment:
Should this test also test that `onClick` was called with specific parameter values?

Reply:
Good call. I updated to include parameters. With RTL, I couldn't specify a mock event like before (`userEvent.click` doesn't allow for this parameter type to get passed). I used `expect.anything()` for that param because I couldn't find docs and approached the same frustration levels as this [SO OP](https://stackoverflow.com/questions/63735881/react-jest-testing-mock-a-click-event-object).